accessibilityTest/AccessibilityFacet.ts

```diff
@@ -24,38 +25,72 @@ export const AccessibilityFacet = () => {
       done();
     });
 
-    it('search should be accessible', async done => {
-      const facetElement = getFacetElement();
-      getFacetColumn().appendChild(facetElement.el);
-      await afterDeferredQuerySuccess();
-      (get(facetElement.el) as Facet).facetSearch.focus();
-      await afterDelay(1000);
-      const axeResults = await axe.run(getRoot());
-      expect(axeResults).toBeAccessible();
-      done();
-    });
+    describe('with facet element', () => {
+      let facetElement: Dom;
 
-    it('should still be accessible when search has been opened', async done => {
-      const facetElement = getFacetElement();
-      getFacetColumn().appendChild(facetElement.el);
-      await afterDeferredQuerySuccess();
-      (get(facetElement.el) as Facet).facetSearch.focus();
-      await afterDelay(1000);
-      (get(facetElement.el) as Facet).facetSearch.dismissSearchResults();
-      await afterDelay(1000);
-      const axeResults = await axe.run(getRoot());
-      expect(axeResults).toBeAccessible();
-      done();
-    });
+      function getMagnifierSVG() {
+        return facetElement.el.querySelector<HTMLElement>('.coveo-facet-search-magnifier-svg');
+      }
 
-    it('settings should be accessible', async done => {
-      const facetElement = getFacetElement();
-      getFacetColumn().appendChild(facetElement.el);
-      await afterDeferredQuerySuccess();
-      facetElement.find('.coveo-facet-header-settings').click();
-      const axeResults = await axe.run(getRoot());
-      expect(axeResults).toBeAccessible();
-      done();
```

Comment:
Those tests weren't removed, they were just moved. Their first few lines up to the first `afterDelay` (or `afterDeferredQuerySuccess`) were moved to a `beforeEach`.